src/constants/FilterTypes.js

```diff
@@ -0,0 +1,3 @@
+// FILTTER ACTIONS
```

Comment:
Can you rename the file to `FilterActionTypes` instead ?

Reply:
All files under constants are ItemTypes, ListTypes I put the name FilterTypes in order to follow the pattern. If I were to change it it would break the pattern.